src/ini/config.c

```diff
@@ -0,0 +1,81 @@
+#include "ini.h"
+
+typedef struct /*Can be changed in the future*/
+{
+    const char* orchestratorIP;
+    const char* ochestratorPort;
+} configuration_node;
+
+typedef struct
+{
+    const char* port;
+    const char* expectedNodes;
+} configuration_orch;
+
+/*
+This function will be used in ini_parse() for node ini file parsing.
+If 0 return that means that the ini file is not in the right schem 
+*/
+static int handler_node(void* user, const char* section, const char* name,
+                   const char* value) {
+
+    configuration_node* pconfig = (configuration_node*)user;
+
+    #define MATCH(s, n) strcmp(section, s) == 0 && strcmp(name, n) == 0
+    if (MATCH("Configuration", "OchestratorPort")) {
+        pconfig->ochestratorPort = strdup(value);
+    } else if (MATCH("Configuration", "OrchestratorIP")) {
+        pconfig->orchestratorIP = strdup(value);
+    } else {
+        return 0;  /* unknown section/name, error */
```

Comment:
Isn't `0` regarded as "success" and `1` as "failure"?

Reply:
Actually 1 is `true` and 0 is `false`, that is opposed to program return codes in UNIX where 0 is success